content/en/news/releases/1.19.x/announcing-1.19/change-notes/index.md

```diff
@@ -0,0 +1,302 @@
+
+- **Added** values to the Istio Pilot Helm charts for configuring additional container arguments, `volumeMounts`, and `volumes`. Can be used in conjunction with cert-manager `istio-csr`.
+  ([Issue #113](https://github.com/cert-manager/istio-csr/issues/113))
+
+- **Added** values to the Istiod Helm chart for configuring [topologySpreadConstraints](https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/) on the Deployment. Can be used for better placement of istiod workloads.
+  ([Issue #42938](https://github.com/istio/istio/issues/42938))
+
+- **Added** Allow setting `terminationGracePeriodSeconds` for ztunnel pod via Helm chart.
+
+- **Fixed** an issue where removing field(s) from IstioOperator and re-installing did not reflect changes in existing IstioOperator spec.  ([Issue #42068](https://github.com/istio/istio/issues/42068))
+
+- **Fixed** `ValidatingWebhookConfiguration` not being generated correctly with operator installation when the revision is not set.
+  ([Issue #43893](https://github.com/istio/istio/issues/43893))
+
+- **Fixed** an issue where the operator did not reject invalid CIDR entries that included spaces.
+  ([Issue #45338](https://github.com/istio/istio/issues/45338))
+
+- **Fixed** an issue where the hostname package is not listed as a dependency for the VM packages.
+  ([Issue #45866](https://github.com/istio/istio/issues/45866))
+
+- **Fixed** an issue preventing the gateway chart from being used with a custom `HorizontalPodAutoscaler` resource.
+
+- **Fixed** an issue that Istio should using `IMDSv2` as possible on AWS.
+  ([Issue #45825](https://github.com/istio/istio/issues/45825))
+
+- **Fixed** OpenShift profile setting `sidecarInjectorWebhook` causing `k8s.v1.cni.cncf.io/networks` to be overwritten when using multiple networks.
+  ([Issue #43632](https://github.com/istio/istio/issues/43632)),([Issue #45034](https://github.com/istio/istio/issues/45034))
+
+- **Fixed** Null traversal issue when using `datadog` or `stackdriver` with no tracing options.
+  ([Issue #45855](https://github.com/istio/istio/issues/45855))
+
+- **Fixed** an issue preventing the ports of waypoint and ztunnel ports being exposed. Now scrape configuration files can be created for the Ambient components too.
+  ([Issue #45093](https://github.com/istio/istio/issues/45093))
+
+- **Removed** the following experimental `istioctl` commands: `add-to-mesh`, `remove-from-mesh` and `kube-uninject`.
+Usage of automatic sidecar injection is recommended instead.
+
+- **Removed** the `ENABLE_LEGACY_FSGROUP_INJECTION` feature flag. This was intended to support Kubernetes 1.18 and older, which are out of support.
+
+- **Removed** obsolete manifests from the `base` Helm chart. See Upgrade Notes for more information.
+
+## istioctl
+
+- **Improved** IST0123 warning message description.
+
+- **Updated** minimum supported Kubernetes version to `1.25.x`.
```

Comment:
I think this can be removed since we already have a supported version list, and in the past it wasn't updated this on time so we needed a release note.

Reply:
I agree this should be removed. @ericvn thoughts? We don't typically include these statements as they're on the supported releases page.